Resources/Locale/en-US/advertisements/vending/donut.ftl

```diff
@@ -0,0 +1,4 @@
+﻿advertisement-donut-1 = Homer Simpson approves!
```

Comment:
this probably doesn't matter that much but generally we want to stay copyright safe and avoid references to intellectual properties.